osi_environment.proto

```diff
@@ -324,4 +336,150 @@ message EnvironmentalConditions
+        {
+            // Fractional cloud cover level is unknown (must not be used in ground truth).
+            //
+            FRACTIONAL_CLOUD_COVER_UNKNOWN = 0;
+
+            // Other (unspecified but known) fractional cloud cover level.
+            //
+            FRACTIONAL_CLOUD_COVER_OTHER = 1;
+
+            // 0/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_ZERO_OKTAS = 2;
+
+            // 1/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_ONE_OKTAS = 3;
+
+            // 2/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_TWO_OKTAS = 4;
+
+            // 3/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_THREE_OKTAS = 5;
+
+            // 4/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_FOUR_OKTAS = 6;
+
+            // 5/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_FIVE_OKTAS = 7;
+
+            // 6/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_SIX_OKTAS = 8;
+
+            // 7/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_SEVEN_OKTAS = 9;
+
+            // 8/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_EIGHT_OKTAS = 10;
+
+            // Sky obscured, describes situations where the sky is not perceivable, e.g. in dense fog.
+            //
+            FRACTIONAL_CLOUD_COVER_SKY_OBSCURED = 11;
+        }
+    }
+
+    //
+    // \brief Defines wind properties.
+    //
+    message Wind
+    {
+        // The origin direction of the wind (not the target direction) in the ground/xy-plane of the
+        // world coordinate system. Corresponds to the heading/yaw angle, counted counterclockwise.
+        // 0 pointing north. If north is not explicitly defined via \c #osi3::GroundTruth::proj_string,
+        // then north is pointing in y-axis direction. The default orientation (x, y, z) is easting, northing, up [1].
+        //
+        // \note 0: north; +pi/2: west; pi: south, 3/2 pi: east.
+        //
+        // Unit: rad
+        //
+        // The preferred angular range is [0, 2pi].
+        //
+        // \par References:
+        // [1] PROJ contributors. (2019). <em>PROJ coordinate transformation software library</em>. Open Source Geospatial Foundation. Retrieved January 25, 2019, from https://proj.org/usage/projections.html
+        //
+        optional double origin_direction = 1;
+
+        // The wind speed.
+        //
+        // Unit: m/s
+        //
+        // \rules
+        // is_greater_than_or_equal_to: 0
+        // \endrules
+        //
+        optional double speed = 2;
+    }
+
+    //
+    // \brief Properties of the sun.
+    //
+    message Sun
+    {
+        // Azimuth of the sun, counted counterclockwise.
```

Comment:
**Output CCB 13.04.2022:**
1. @HendrikAmelunxen Update _azimuth_ and _elevation_ orientation description by adding the method that is used to define the positive direction (e.g., using the right hand rule).